src/components/ArticleManager.jsx

```diff
@@ -0,0 +1,28 @@
+import { useState, useEffect } from "react";
+
+import ArticlesList from "./ArticlesList";
+import PostArticle from "./PostArticle";
+import SortArticles from "./SortArticles";
+
+import { getArticles } from "../api/articles";
+
+function ArticlesManager() {
+  const [articles, setArticles] = useState([]);
```

Comment:
Could we have stored this state any lower? We are using it in articlesList currently and isn't being sent anywhere else.